apps/src/lib/ui/accounts/DeleteAccountHelpers.jsx

```diff
@@ -52,50 +53,41 @@ const CHECKBOX_MAP = {
     checked: false,
     label: (
       <span>
-        <strong>{i18n.deleteAccountDialog_checkbox1_1()}</strong>
-        {i18n.deleteAccountDialog_checkbox1_2()}
-        <a href={RELEASE_OR_DELETE_RECORDS_EXPLANATION} target="_blank">
-          {i18n.learnMore()}
-        </a>
```

Comment:
is it intentional that this "learn more" link went away?

Reply:
Great find!! Thanks